src/hooks/useDynamicBottomSheet.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {useBottomSheet, BottomSheetProps} from '@ui/library';
+
+type DynamicBottomSheetProps = Omit<BottomSheetProps, 'children'>;
+
+export function useDynamicBottomSheet() {
```

Comment:
What I had in mind was to implement this in the `useBottomSheet` hook so we have a single component.

The logic could be the following:

```
...
 <BottomSheetView>{bottomSheetContent ?? children}</BottomSheetView>
...

function openBottomSheet(content?: React.ReactNode) {
   if (content) {
     setBottomSheetContent(content);
   }
   bottomSheetRef.current?.expand();
}
```

or we could expose another function like `updateBottomSheetContent(content)`